balboa-client-kafka/src/main/java/com/socrata/balboa/impl/MetricLoggerJava.java

```diff
@@ -63,4 +63,9 @@ private MetricLoggerJava(String topic) {
     public void create(IdParts entity, IdParts name, long value, long timestamp, Metric.RecordType recordType) {
         this.logger.logMetric(entity.toString(), name.toString(), value, timestamp, recordType);
     }
+
+    @Override
+    public void close() throws Exception {
+        // TODO Don't know how or what it means to shut this down.
```

Comment:
If you have to override the close method (it's abstract or from an interface), the comment should read something like "There is no cleanup activity to be completed."

As long as you have closed any member variables of this class instance, and freed any resources allocated to this class instance, close has accomplished it's objective.